src/components/[guild]/EditGuild/EditGuildDrawer.tsx

```diff
@@ -97,6 +99,11 @@ const EditGuildDrawer = ({
     socialLinks,
     featureFlags: isSuperAdmin ? featureFlags : undefined,
     tags: savedTags,
+    eventSources: {
+      EVENTBRITE: eventSources.EVENTBRITE,
+      LINK3: "",
```

Comment:
Is this intended? Or should it be `eventSources.LINK3`?
Edit: now I see, so we don't support LINK3 yet?